examples/http/tracer_common.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+#include "opentelemetry/exporters/ostream/span_exporter.h"
+#include "opentelemetry/sdk/trace/simple_processor.h"
+#include "opentelemetry/sdk/trace/tracer_provider.h"
+#include "opentelemetry/trace/provider.h"
+
+#include "opentelemetry/context/propagation/global_propagator.h"
+#include "opentelemetry/context/propagation/text_map_propagator.h"
+#include "opentelemetry/trace/propagation/http_trace_context.h"
+
+#include <cstring>
+#include <iostream>
+#include <vector>
+#include "opentelemetry/ext/http/client/http_client.h"
```

Comment:
I wonder if we can _eventually_ generalize some of this, since the same / similar logic would be necessary for gRPC metadata. I like how it was handled in dapr examples:

https://docs.dapr.io/developing-applications/building-blocks/observability/w3c-tracing/w3c-tracing-howto/

Reply:
Yes, we should eventually reusable the code for both HTTP and grpc.